dir.props

```diff
@@ -274,6 +274,10 @@
 
   <!-- Setup properties per OSGroup -->
   <Choose>
+    <When Condition="'$(IsReferenceAssembly)' == 'true' OR'$(MSBuildProjectExtension)' == '.pkgproj'">
```

Comment:
Yes I would expect it to always be undefined for reference assemblies coming from facade references https://github.com/dotnet/buildtools/blob/master/src/Microsoft.DotNet.Build.Tasks/PackageFiles/resolveContract.targets#L27